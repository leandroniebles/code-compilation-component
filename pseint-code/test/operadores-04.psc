Proceso sin_titulo
	
	Escribir 3+1
	Escribir +4
	Escribir 2-6
	Escribir -4
	Escribir 2*2
	Escribir 12/3
	Escribir 2^2
	Escribir 10%6
		
	// tienen que dar verdadero
	Escribir 2<3
	Escribir 2<=2
	Escribir 2<=3
	Escribir 2>1
	Escribir 2>=1
	Escribir 2>=2
	Escribir 2=2
	Escribir 2==2
	Escribir 2!=3
	Escribir 2<>3
	Escribir !falso
	Escribir ~falso
	Escribir Verdadero&Verdadero
	Escribir Verdadero&&Verdadero
	Escribir Verdadero|falso
	Escribir Verdadero||falso
	Escribir falso|Verdadero
	Escribir falso||Verdadero
	Escribir Verdadero|Verdadero
	Escribir Verdadero||Verdadero
	
	// tienen que dar falso
	Escribir 2<1
	Escribir 2<2
	Escribir 2<=1
	Escribir 2>2
	Escribir 2>3
	Escribir 2>=3
	Escribir 2=3
	Escribir 2==3
	Escribir 2!=2
	Escribir 2<>2
	Escribir !verdadero
	Escribir ~verdadero
	Escribir Verdadero&falso
	Escribir Verdadero&&falso
	Escribir falso&Verdadero
	Escribir falso&&Verdadero
	Escribir falso&falso
	Escribir falso&&falso
	Escribir falso|falso
	Escribir falso||falso
	
	
FinProceso
