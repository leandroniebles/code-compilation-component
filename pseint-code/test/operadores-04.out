4
4
-4
-4
4
4
4
4
VERDADERO
VERDADERO
VERDADERO
VERDADERO
VERDADERO
VERDADERO
VERDADERO
VERDADERO
VERDADERO
VERDADERO
VERDADERO
VERDADERO
VERDADERO
VERDADERO
VERDADERO
VERDADERO
VERDADERO
VERDADERO
VERDADERO
VERDADERO
FALSO
FALSO
FALSO
FALSO
FALSO
FALSO
FALSO
FALSO
FALSO
FALSO
FALSO
FALSO
FALSO
FALSO
FALSO
FALSO
FALSO
FALSO
FALSO
FALSO
